doc/template/HOW-TO.md

```diff
@@ -0,0 +1,128 @@
+## Setup
+
+I start cloning our example
+
+    $ git clone -b gh-pages https://github.com/swcarpentry/lesson-example.git
```

Comment:
And for notes about editing the template, I think I'd rather have folks clone lesson-template, and then add `lesson-example` as a remote named `example` (or some such).